### DISCUSSION TITLE
No documents are valid to register in GitHub Student

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17820] [GitHub Education]
Hi!
I’m trying to register in Github Student Program. I’m 15 and I frequent an Italian School. I don’t know wich document I can to send you because it doesn’t exist a current enrollment document. I sent you a screenshot of my electronic register and of my evaluation document and you don’t accept.
When I upload the document it says me “Your image should include the name of your school” but the school name is visible!

I contact the support but they answered me with a precompiled answer that didn’t help me.
Can you help? Thanks

**Comment by mlarasusan** [comment]
Hello 👋 
[Here](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) you can find more information on how to get the student developer pack :] Also, check [this](https://github.com/github-community/community/discussions/31#discussioncomment-2561585)!

**Comment by northdotcom** [comment]
Hai per caso un libretto delle giustificazioni cartaceo? quello potrebbe aiutare. Altrimenti magari la poste pay dello studente oppure un tesserino della scuola (se lo hai)

**Reply by ghost** [reply]
Nessuna di queste. Ho inviato uno screenshot del registro elettronico ma non lo accettano

**Reply by northdotcom** [reply]
hai chiesto alla tua scuola? magari loro hanno qualcosa da darti. Chiedi ad un professore, possibilmente vice preside o preside. Magari puoi anche convincerli ad usare github for teachers. Non so che indirizzo fai ma ci saranno sicuramente delle ore di informatica

**Reply by ghost** [reply]
Proverò... grazie
